scripts/calculateRevenue/protocols/aave/types.ts

```diff
@@ -0,0 +1,20 @@
+import { Address } from 'viem'
+
+export interface BalanceSnapshot {
+  scaledATokenBalance: bigint
+  liquidityIndex: bigint
+  timestamp: number
+}
+
+export interface ReserveFactor {
+  reserveFactor: bigint
+  timestamp: number
+}
+
+export interface ReserveData {
+  reserveTokenAddress: Address
+  reserveTokenDecimals: number
+  aTokenAddress: Address
+  liquidityIndex: bigint
+  reserveFactor: bigint
+}
```

Comment:
It would be helpful to explain the fields, possibly linking to aave docs.
So next time someone needs to look at this their work will be facilitated.